cves/CVE-2018-6912.yml

```diff
@@ -64,7 +64,14 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  When parsing audio-visual data for Ut Video Codec files, a fast and lossless video codec, 
+  no check is done for the length and integrity of the inputted file. Should this file 
+  change in size during the decoding process or be carefully crafted such that the format 
+  is against specifications, an out of array read occurs, leading to a potential 
+  Denial of Service. 
+  Attackers could craft files empty or variable length files leading to crashes, 
+  threatening the integrity of any system running FFmpeg.
 bounty_instructions: |
```

Comment:
Good job explaining how this vulnerability could lead to a denial of service attack, it could also damage the system's data integrity.

Reply:
Thanks! Clarified in d47de23